charts/celeborn/templates/configmap.yaml

```diff
@@ -29,11 +29,12 @@ metadata:
 data:
   celeborn-defaults.conf: |-
     {{- $namespace := .Release.Namespace }}
-    celeborn.ha.enabled=true
     celeborn.master.endpoints={{ range until (.Values.masterReplicas |int) }}celeborn-master-{{ . }}.celeborn-master-svc.{{ $namespace }}.svc.{{ $.Values.cluster.name }}.local,{{ end }}
     {{- range until (.Values.masterReplicas |int) }}
     celeborn.ha.master.node.{{ . }}.host=celeborn-master-{{ . }}.celeborn-master-svc.{{ $namespace }}.svc.{{ $.Values.cluster.name }}.local
     {{- end }}
+    celeborn.ha.master.ratis.raft.server.storage.dir:{{ .Values.volumes.master.ratis.path }}
```

Comment:
the separator between key/value  need to be "=" rather than ":"？

Reply:
Sure, fixed and also modify `volumes` from string to array.